features/request_construction/basic_auth.feature

```diff
@@ -5,7 +5,7 @@ Feature: Adding Basic Auth
       """
       {"Authorization":"Basic dXNlcjpwYXNz"}
       """
-    And expected GET sent to `/query` with 
```

Comment:
Was this previously mapped in rest_steps?

Reply:
This was previously sloppily matched due to the missing anchor here: https://github.com/brightcove/brine/pull/194/files#diff-688e9e1d81dcfc63a49490439ed1b02fL124